packit_service/service/app.py

```diff
@@ -46,8 +47,12 @@ def get_flask_application():
     app.register_blueprint(builds_blueprint)
     s = ServiceConfig.get_service_config()
     # https://flask.palletsprojects.com/en/1.1.x/config/#SERVER_NAME
+    # also needs to contain port if it's not 443
     app.config["SERVER_NAME"] = s.server_name
     app.config["PREFERRED_URL_SCHEME"] = "https"
+    if getenv("DEPLOYMENT") in ("dev", "stg"):
+        app.config["DEBUG"] = True
+    app.logger.setLevel(logging.DEBUG)
```

Comment:
```suggestion
        app.logger.setLevel(logging.DEBUG)
```

Reply:
I'd set it to debug no matter what - I actually wasn't getting any log messages from flask/werkzeug to be honest